pallets/roles/src/tests.rs

```diff
@@ -0,0 +1,92 @@
+// This file is part of Webb.
+// Copyright (C) 2022 Webb Technologies Inc.
```

Comment:
Change for all files
```suggestion
// Copyright (C) 2022-2023 Webb Technologies Inc.
```